modules/acs-architecture-overview.adoc

```diff
@@ -3,102 +3,46 @@
+* *Scanner*:
+Scanner is a Red Hat-developed and certified vulnerability scanner for scanning container images and their associated databases.
+It analyzes all image layers to check known vulnerabilities from the Common Vulnerabilities and Exposures (CVEs) list.
 Scanner also identifies vulnerabilities in packages installed by package managers and in dependencies for multiple programming languages.
-//TODO: Add link to Examine images topic.
-
-[NOTE]
-====
-Scanner only scans those images that are not already scanned by other integrated vulnerability scanners.
-It means that if you have integrated {product-title} with other vulnerability scanners, Scanner checks and uses the scanning results from the integrated scanner if available.
-====
-//TODO: Add link to integrate-with-vulnerability-scanners topic
-
-[id="per-cluster-components_{context}"]
-== Per-cluster components
-
-You deploy the per-cluster components into each cluster that you want to monitor.
-{product-title} includes the following per-cluster components:
-
-* Sensor
-* Admission controller
 
 [discrete]
-=== Sensor
-
-{product-title} uses the Sensor component to monitor Kubernetes and {ocp} clusters.
-It handles interactions with the {ocp} or Kubernetes API server for policy detection and enforcement, and it coordinates with Collector.
+== Secured cluster services
+You install the secured cluster services on each cluster that you want to secure by using the {product-title} (*Cluster N* in _Figure 1_ and _2_). Cluster where you install Central is also secured and includes these services.
 
-[discrete]
-=== Admission controller
+* *Sensor*:
+Sensor is the service responsible for analyzing and monitoring the cluster.
+It handles interactions with the OpenShift Container Platform or Kubernetes API server for policy detection and enforcement, and it coordinates with Collector.
 
-The admission controller prevents users from creating workloads that violate security policies in {product-title}.
-//TODO: Add link to admission-controller-enforcement.
+* *Admission controller*:
+The admission controller prevents users from creating workloads that violate security policies in {product-title-short}.
 
-[id="per-node-components_{context}"]
-== Per-node components
+* *Collector*:
+Collector analyzes and monitors container activity on cluster nodes.
+It collects information about container runtime and network activity and sends the collected data to Sensor.
 
-You deploy the per-node components in all nodes that you want to monitor.
-{product-title} includes the following per-cluster components:
+* *Scanner* (only on {ocp}):
+On {ocp}, RHACS installs a lightweight version of Scanner on each secured cluster (_Figure 1_) to enable scanning of images in the integrated OCR.
 
-* Collector
-
-[discrete]
-=== Collector
+Figure 2 depicts the architecture in a non-OCP Kubernetes environment, where you only install the Scanner centrally.
```

Comment:
```suggestion
Figure 2 depicts the architecture in a non-OCP Kubernetes environment, where you install only the Scanner centrally.
```

Please double-check your use of "only" throughout. It's a subtle thing, but "only" is supposed to go close to the thing you're limiting.